content/en/docs/tasks/security/cert-management/custom-ca-k8s/index.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Istio Custom Certificate Authority Integration using Kubernetes CSR API [experimental]
+description: Shows how to use a Custom Certificate Authority (that integrates with the Kubernetes CSR API) to provision Istio workload certificates.
+weight: 90
```

Comment:
This controls where in the list of documents under the same folder, this one should go. I think it should be last, so higher than the other highest one (90).
```suggestion
weight: 100
```

Reply:
Agreed